.github/workflows/cross-platform-test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     timeout-minutes: 20
     strategy:
       matrix:
-        os: [macOS-latest, windows-latest]
+        os: [windows-latest]
```

Comment:
[F] By removing the less valuable `macOS-latest` tests we make get back more minutes than we expend running the E2E tests in the main workflow.